core/src/impl/Kokkos_QuadPrecisionMath.hpp

```diff
@@ -72,22 +72,16 @@ namespace Experimental {
 
 // clang-format off
 // Numeric distinguished value traits
-// Workaround GCC bug https://godbolt.org/z/qWb5oe4dx
-// error: '__builtin_huge_valq()' is not a constant expression
-#if defined(KOKKOS_COMPILER_GNU) && (KOKKOS_COMPILER_GNU >= 710)
```

Comment:
We were previously only defining these guarded traits for `defined(KOKKOS_COMPILER_GNU)` - I take it that was a pre-existing mistake, and should have been `!defined(GNU) || GNU>=710`?

Reply:
That's what I think.

Reply:
Yes